*** ./build/test-testfiles/config-bibtex/10-bibtex-inline.tlg	2022-05-13 22:20:38.385929294 +0800
--- ./build/test-testfiles/config-bibtex/10-bibtex-inline.xetex.log	2022-05-13 22:20:45.129875757 +0800
***************
*** 16,21 ****
--- 16,23 ----
  Completed box being shipped out [1]
  \vbox(710.18088+0.0)x439.87962
  .\hbox(0.0+0.0)x0.0
+ ..\special{pdfcolorstackinit 1 page direct (0 g 0 G)}
+ .\hbox(0.0+0.0)x0.0
  ..\kern -72.26999
  ..\vbox(0.0+0.0)x0.0, glue set 72.26999fil
  ...\kern -72.26999
***************
*** 38,43 ****
--- 40,47 ----
  ..\vbox(674.33032+0.0)x426.79135, glue set 381.23964fil
  ...\write-{}
  ...\special{color push gray 0}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnotehinttrue }{}}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnote@aux }[2]{}}
  ...\glue(\topskip) 2.58081
  ...\hbox(9.41919+2.144)x426.79135, glue set 298.3194fil
  ....\hbox(0.0+0.0)x24.09
***************
*** 74,80 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.00568
  ...\hbox(10.92532+2.144)x426.79135, glue set 285.7744fil
  ....\hbox(0.0+0.0)x24.09
--- 78,83 ----
***************
*** 118,124 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.72862
  ...\hbox(9.20238+2.144)x426.79135, glue set 334.45439fil
  ....\hbox(0.0+0.0)x24.09
--- 121,126 ----
***************
*** 142,148 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.00568
  ...\hbox(10.92532+2.144)x426.79135, glue set 324.92064fil
  ....\hbox(0.0+0.0)x24.09
--- 144,149 ----
***************
*** 174,180 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.72862
  ...\hbox(9.20238+2.144)x426.79135, glue set 319.39815fil
  ....\hbox(0.0+0.0)x24.09
--- 175,180 ----
***************
*** 202,208 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.00568
  ...\hbox(10.92532+2.144)x426.79135, glue set 309.8644fil
  ....\hbox(0.0+0.0)x24.09
--- 202,207 ----
***************
*** 238,244 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.78857
  ...\hbox(8.14243+2.61375)x426.79135, glue set 288.11728fil
  ....\hbox(0.0+0.0)x24.09
--- 237,242 ----
***************
*** 268,274 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.53592
  ...\hbox(10.92532+2.61375)x426.79135, glue set 278.58353fil
  ....\hbox(0.0+0.0)x24.09
--- 266,271 ----
***************
*** 304,310 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.31882
  ...\hbox(8.14243+1.879)x426.79135, glue set 327.26352fil
  ....\hbox(0.0+0.0)x24.09
--- 301,306 ----
***************
*** 328,334 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.27068
  ...\hbox(10.92532+1.879)x426.79135, glue set 317.72977fil
  ....\hbox(0.0+0.0)x24.09
--- 324,329 ----
***************
*** 358,364 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 10.05357
  ...\hbox(8.14243+1.879)x426.79135, glue set 308.87082fil
  ....\hbox(0.0+0.0)x24.09
--- 353,358 ----
***************
*** 385,391 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.27068
  ...\hbox(10.92532+1.879)x426.79135, glue set 299.33707fil
  ....\hbox(0.0+0.0)x24.09
--- 379,384 ----
***************
*** 418,424 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.99362
  ...\hbox(9.20238+2.144)x426.79135, glue set 334.45439fil
  ....\hbox(0.0+0.0)x24.09
--- 411,416 ----
***************
*** 442,448 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.72862
  ...\hbox(9.20238+2.144)x426.79135, glue set 324.4209fil
  ....\hbox(0.0+0.0)x24.09
--- 434,439 ----
***************
*** 475,481 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.72862
  ...\hbox(9.20238+2.144)x426.79135, glue set 324.4209fil
  ....\hbox(0.0+0.0)x24.09
--- 466,471 ----
***************
*** 507,516 ****
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue -2.144
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
--- 497,505 ----
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -2.144
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
***************
*** 531,536 ****
--- 520,526 ----
  ...\glue 0.0 plus 1.0fil minus 1.0fil
  .\kern 710.18088
  .\kern 0.0
+ .\kern 0.0
  Package fontspec Info: Font family 'FandolHei(0)' created for font 'FandolHei'
  (fontspec)             with options
  (fontspec)             [Extension={.otf},UprightFont={*-Regular},BoldFont={*-Bold}].
***************
*** 582,587 ****
--- 572,578 ----
  ...\write-{}
  ...\write1{\@writefile{toc}{\protect \contentsline {chapter}{参考文献}{\thepage \ET
  C.}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\glue(\topskip) 12.0
  ...\rule(0.0+0.0)x*
***************
*** 589,595 ****
  ...\glue 15.10124 plus -1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(12.6071+3.01927)x426.79135, glue set 181.27568fil
  ....\glue(\leftskip) 0.0 plus 1.0fil
  ....\TU/FandolHei(0)/m/n/16.06 参
--- 580,585 ----
***************
*** 607,613 ****
  ...\penalty 10000
  ...\glue 27.10124 plus -1.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\hbox(8.24179+1.97084)x398.33861, glue set 0.15424, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
  .....\glue 0.0
--- 597,602 ----
***************
*** 746,752 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.016
  ...\hbox(8.17854+1.86545)x398.33861, glue set 155.08296fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 735,740 ----
***************
*** 822,828 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.99615
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.21777, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 810,815 ----
***************
*** 1028,1034 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.48903
  ...\hbox(8.2734+1.97084)x398.33861, glue set 103.55951fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 1015,1020 ----
***************
*** 1116,1122 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.79468
  ...\hbox(8.29448+1.91815)x398.33861, glue set 90.3853fil, shifted 28.45274
  ....\hbox(7.25108+1.64412)x0.0
--- 1102,1107 ----
***************
*** 1194,1200 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.92114
  ...\hbox(8.2207+1.9287)x398.33861, glue set 0.09877, shifted 28.45274
  ....\hbox(7.20892+1.64412)x0.0
--- 1179,1184 ----
***************
*** 1321,1327 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.99615
  ...\hbox(7.1984+2.29756)x398.33861, glue set - 0.4655, shifted 28.45274
  ....\hbox(6.97707+1.64412)x0.0
--- 1305,1310 ----
***************
*** 1447,1453 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.56404
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.24342, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 1430,1435 ----
***************
*** 1616,1622 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.56404
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.45659, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 1598,1603 ----
***************
*** 1724,1730 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.52065
  ...\hbox(8.24179+1.94977)x398.33861, glue set 0.01776, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 1705,1710 ----
***************
*** 1849,1855 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.37556
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.76987, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 1829,1834 ----
***************
*** 1985,1991 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.56404
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.24365, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 1964,1969 ----
***************
*** 2108,2114 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.57457
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.66132, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 2086,2091 ----
***************
*** 2242,2248 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.37556
  ...\hbox(7.1984+2.29756)x398.33861, glue set - 0.10126, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 2219,2224 ----
***************
*** 2388,2394 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.51135
  ...\hbox(7.25108+2.28703)x398.33861, glue set - 0.00581, shifted 28.45274
  ....\hbox(7.25108+1.64412)x0.0
--- 2364,2369 ----
***************
*** 2521,2527 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.51135
  ...\hbox(7.25108+2.29756)x398.33861, glue set 0.42717, shifted 28.45274
  ....\hbox(7.20892+1.64412)x0.0
--- 2496,2501 ----
***************
*** 2637,2643 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.62851
  ...\hbox(8.28395+1.93922)x398.33861, glue set 59.64197fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 2611,2616 ----
***************
*** 2733,2739 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.8052
  ...\hbox(8.31557+1.90761)x398.33861, glue set - 0.76524, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 2706,2711 ----
***************
*** 2849,2855 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.7209
  ...\hbox(8.43149+2.10786)x398.33861, glue set 137.5279fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 2821,2826 ----
***************
*** 3178,3184 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.69174
  ...\hbox(8.2207+1.85492)x398.33861, glue set 0.325, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 3149,3154 ----
***************
*** 3317,3323 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.016
  ...\hbox(8.17854+1.9287)x398.33861, glue set 0.28651, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 3287,3292 ----
***************
*** 3485,3491 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.96454
  ...\hbox(7.1984+2.29756)x398.33861, glue set - 0.71106, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 3454,3459 ----
***************
*** 3584,3590 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.57457
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.12486, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 3552,3557 ----
***************
*** 3690,3696 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.33217
  ...\hbox(8.24179+1.94977)x398.33861, glue set - 0.73991, shifted 28.45274
  ....\hbox(7.25108+1.64412)x0.0
--- 3657,3662 ----
***************
*** 3799,3805 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.9013
  ...\hbox(7.20892+2.29756)x398.33861, glue set - 0.9919, shifted 28.45274
  ....\hbox(7.20892+1.64412)x0.0
--- 3765,3770 ----
***************
*** 3942,3948 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.5969
  ...\hbox(8.23125+1.91815)x398.33861, glue set 0.17368, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 3907,3912 ----
***************
*** 4076,4082 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.07922
  ...\hbox(8.19963+1.86545)x398.33861, glue set 98.89056fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 4040,4045 ----
***************
*** 4167,4173 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.99492
  ...\hbox(8.19963+1.88654)x398.33861, glue set 23.36545fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 4130,4135 ----
***************
*** 4264,4270 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.97507
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.06883, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 4226,4231 ----
***************
*** 4384,4390 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.34271
  ...\hbox(8.23125+1.98138)x398.33861, glue set 0.08244, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 4345,4350 ----
***************
*** 4639,4645 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.53117
  ...\hbox(8.23125+2.28705)x398.33861, glue set 0.39548, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 4599,4604 ----
***************
*** 4846,4852 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.62973
  ...\hbox(7.1984+2.29756)x398.33861, glue set 0.24536, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 4805,4810 ----
***************
*** 5046,5052 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.56403
  ...\hbox(7.1984+2.29756)x398.33861, glue set 8.34685fil, shifted 28.45274
  ....\hbox(7.12462+1.64412)x0.0
--- 5004,5009 ----
***************
*** 5114,5123 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue 0.0
  ...\glue -2.29756
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
--- 5071,5079 ----
  ...\penalty -51
  ...\glue 0.0
  ...\glue 0.0
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -2.29756
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
