*** ./build/test-testfiles/config-bibtex/10-bibtex-author-year.tlg	2022-05-13 22:20:32.753973956 +0800
--- ./build/test-testfiles/config-bibtex/10-bibtex-author-year.xetex.log	2022-05-13 22:20:38.373929389 +0800
***************
*** 16,21 ****
--- 16,23 ----
  Completed box being shipped out [1]
  \vbox(710.18088+0.0)x439.87962
  .\hbox(0.0+0.0)x0.0
+ ..\special{pdfcolorstackinit 1 page direct (0 g 0 G)}
+ .\hbox(0.0+0.0)x0.0
  ..\kern -72.26999
  ..\vbox(0.0+0.0)x0.0, glue set 72.26999fil
  ...\kern -72.26999
***************
*** 38,43 ****
--- 40,47 ----
  ..\vbox(674.33032+0.0)x426.79135, glue set 60.07399fil
  ...\write-{}
  ...\special{color push gray 0}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnotehinttrue }{}}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnote@aux }[2]{}}
  ...\glue(\topskip) 3.85757
  ...\hbox(8.14243+2.13194)x426.79135, glue set 277.42133fil
  ....\hbox(0.0+0.0)x24.09
***************
*** 63,69 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.6561
  ...\hbox(8.28696+2.13194)x426.79135, glue set 251.23552fil
  ....\hbox(0.0+0.0)x24.09
--- 67,72 ----
***************
*** 98,104 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.71632
  ...\hbox(8.22673+2.13194)x426.79135, glue set 263.70209fil
  ....\hbox(0.0+0.0)x24.09
--- 101,106 ----
***************
*** 133,139 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.6561
  ...\hbox(8.28696+2.13194)x426.79135, glue set 251.23552fil
  ....\hbox(0.0+0.0)x24.09
--- 135,140 ----
***************
*** 168,174 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.01773
  ...\hbox(10.92532+2.13194)x426.79135, glue set 241.70177fil
  ....\hbox(0.0+0.0)x24.09
--- 169,174 ----
***************
*** 209,215 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.6561
  ...\hbox(8.28696+2.13194)x426.79135, glue set 248.22427fil
  ....\hbox(0.0+0.0)x24.09
--- 209,214 ----
***************
*** 247,253 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.01773
  ...\hbox(10.92532+2.13194)x426.79135, glue set 238.69052fil
  ....\hbox(0.0+0.0)x24.09
--- 246,251 ----
***************
*** 291,297 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.6561
  ...\hbox(8.28696+2.13194)x426.79135, glue set 229.83157fil
  ....\hbox(0.0+0.0)x24.09
--- 289,294 ----
***************
*** 333,339 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.01773
  ...\hbox(10.92532+2.13194)x426.79135, glue set 220.29782fil
  ....\hbox(0.0+0.0)x24.09
--- 330,335 ----
***************
*** 381,387 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.6561
  ...\hbox(8.28696+2.13194)x426.79135, glue set 216.3532fil
  ....\hbox(0.0+0.0)x24.09
--- 377,382 ----
***************
*** 415,421 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.6561
  ...\hbox(8.28696+2.13194)x426.79135, glue set 213.34195fil
  ....\hbox(0.0+0.0)x24.09
--- 410,415 ----
***************
*** 452,458 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.70454
  ...\hbox(9.23851+2.144)x426.79135, glue set 292.2969fil
  ....\hbox(0.0+0.0)x24.09
--- 446,451 ----
***************
*** 480,486 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.00568
  ...\hbox(10.92532+2.144)x426.79135, glue set 282.76315fil
  ....\hbox(0.0+0.0)x24.09
--- 473,478 ----
***************
*** 516,522 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.69249
  ...\hbox(9.23851+2.144)x426.79135, glue set 286.27441fil
  ....\hbox(0.0+0.0)x24.09
--- 508,513 ----
***************
*** 548,554 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.00568
  ...\hbox(10.92532+2.144)x426.79135, glue set 276.74066fil
  ....\hbox(0.0+0.0)x24.09
--- 539,544 ----
***************
*** 588,594 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.69249
  ...\hbox(9.23851+2.144)x426.79135, glue set 271.21817fil
  ....\hbox(0.0+0.0)x24.09
--- 578,583 ----
***************
*** 624,630 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.00568
  ...\hbox(10.92532+2.144)x426.79135, glue set 261.68442fil
  ....\hbox(0.0+0.0)x24.09
--- 613,618 ----
***************
*** 668,674 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.69249
  ...\hbox(9.23851+2.144)x426.79135, glue set 292.2969fil
  ....\hbox(0.0+0.0)x24.09
--- 656,661 ----
***************
*** 696,702 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.69249
  ...\hbox(9.23851+2.144)x426.79135, glue set 286.27441fil
  ....\hbox(0.0+0.0)x24.09
--- 683,688 ----
***************
*** 728,734 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.64404
  ...\hbox(8.28696+0.26497)x426.79135, glue set 256.27034fil
  ....\hbox(0.0+0.0)x24.09
--- 714,719 ----
***************
*** 760,766 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.52307
  ...\hbox(8.28696+0.26497)x426.79135, glue set 221.38802fil
  ....\hbox(0.0+0.0)x24.09
--- 745,750 ----
***************
*** 791,797 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.52307
  ...\hbox(8.28696+1.69833)x426.79135, glue set 253.2591fil
  ....\hbox(0.0+0.0)x24.09
--- 775,780 ----
***************
*** 826,832 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 10.0897
  ...\hbox(8.28696+1.69833)x426.79135, glue set 218.37677fil
  ....\hbox(0.0+0.0)x24.09
--- 809,814 ----
***************
*** 860,866 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 10.0897
  ...\hbox(8.28696+1.69833)x426.79135, glue set 188.02339fil
  ....\hbox(0.0+0.0)x24.09
--- 842,847 ----
***************
*** 909,915 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 7.45134
  ...\hbox(10.92532+1.69833)x426.79135, glue set 243.72534fil
  ....\hbox(0.0+0.0)x24.09
--- 890,895 ----
***************
*** 950,956 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 10.23424
  ...\hbox(8.14243+0.16861)x426.79135, glue set 332.29834fil
  ....\hbox(0.0+0.0)x24.09
--- 930,935 ----
***************
*** 970,976 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.76396
  ...\hbox(8.14243+2.61375)x426.79135, glue set 271.40688fil
  ....\hbox(0.0+0.0)x24.09
--- 949,954 ----
***************
*** 993,999 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.23451
  ...\hbox(8.22673+0.16861)x426.79135, glue set 294.83441fil
  ....\hbox(0.0+0.0)x24.09
--- 971,976 ----
***************
*** 1021,1027 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.67966
  ...\hbox(8.22673+1.69833)x426.79135, glue set 152.59904fil
  ....\hbox(0.0+0.0)x24.09
--- 998,1003 ----
***************
*** 1060,1066 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 10.23424
  ...\hbox(8.14243+0.16861)x426.79135, glue set 314.23085fil
  ....\hbox(0.0+0.0)x24.09
--- 1036,1041 ----
***************
*** 1079,1085 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.76396
  ...\hbox(8.14243+2.13194)x426.79135, glue set 306.20888fil
  ....\hbox(0.0+0.0)x24.09
--- 1054,1059 ----
***************
*** 1100,1109 ****
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue -2.13194
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
--- 1074,1082 ----
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -2.13194
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
***************
*** 1124,1129 ****
--- 1097,1103 ----
  ...\glue 0.0 plus 1.0fil minus 1.0fil
  .\kern 710.18088
  .\kern 0.0
+ .\kern 0.0
  Package fontspec Info: Font family 'FandolHei(0)' created for font 'FandolHei'
  (fontspec)             with options
  (fontspec)             [Extension={.otf},UprightFont={*-Regular},BoldFont={*-Bold}].
***************
*** 1164,1169 ****
--- 1138,1144 ----
  ...\write-{}
  ...\write1{\@writefile{toc}{\protect \contentsline {chapter}{参考文献}{\thepage \ET
  C.}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\glue(\topskip) 12.0
  ...\rule(0.0+0.0)x*
***************
*** 1171,1177 ****
  ...\glue 15.10124 plus -1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(12.6071+3.01927)x426.79135, glue set 181.27568fil
  ....\glue(\leftskip) 0.0 plus 1.0fil
  ....\TU/FandolHei(0)/m/n/16.06 参
--- 1146,1151 ----
***************
*** 1189,1195 ****
  ...\penalty 10000
  ...\glue 27.10124 plus -1.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\hbox(8.2207+1.88654)x405.71262, glue set 92.19101fil, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
  .....\glue -21.07874
--- 1163,1168 ----
***************
*** 1281,1287 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 5.9106
  ...\hbox(8.26286+1.94977)x405.71262, glue set 119.72336fil, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
--- 1254,1259 ----
***************
*** 1366,1372 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.85915
  ...\hbox(7.25108+2.29756)x405.71262, glue set 8.13249fil, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
--- 1338,1343 ----
***************
*** 1462,1468 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.56404
  ...\hbox(7.1984+2.29756)x405.71262, glue set 125.53052fil, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
--- 1433,1438 ----
***************
*** 1528,1534 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.56404
  ...\hbox(7.1984+1.48604)x405.71262, glue set - 0.41077, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
--- 1498,1503 ----
***************
*** 1689,1695 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.66138
  ...\hbox(7.25108+2.29756)x405.71262, glue set 0.395, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
--- 1658,1663 ----
***************
*** 1820,1826 ****
  ...\penalty -51
  ...\glue 3.01125 plus 3.01125 minus 3.01125
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 6.55351
  ...\hbox(7.20892+2.29756)x405.71262, glue set 0.88461, shifted 21.07874
  ....\hbox(0.0+0.0)x-21.07874
--- 1788,1793 ----
***************
*** 1914,1923 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue 0.0
  ...\glue -0.23186
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
--- 1881,1889 ----
  ...\penalty -51
  ...\glue 0.0
  ...\glue 0.0
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -0.23186
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 22.76228
