*** ./build/test-testfiles/config-crossref/07-3-tables.tlg	2022-05-13 22:19:52.670290502 +0800
--- ./build/test-testfiles/config-crossref/07-3-tables.xetex.log	2022-05-13 22:19:57.126255431 +0800
***************
*** 17,22 ****
--- 17,24 ----
  Completed box being shipped out [3]
  \vbox(710.18088+4.1104)x439.87962
  .\hbox(0.0+0.0)x0.0
+ ..\special{pdfcolorstackinit 1 page direct (0 g 0 G)}
+ .\hbox(0.0+0.0)x0.0
  ..\kern -72.26999
  ..\vbox(0.0+0.0)x0.0, glue set 72.26999fil
  ...\kern -72.26999
***************
*** 82,91 ****
--- 84,98 ----
  ..\vbox(674.33032+0.0)x426.79135, glue set >20000.0fil
  ...\write-{}
  ...\special{color push gray 0}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnotehinttrue }{}}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnote@aux }[2]{}}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\write1{\@writefile{toc}{\protect \contentsline {chapter}{附表清单}{\thepage \ET
  C.}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\glue(\topskip) 12.0
  ...\rule(0.0+0.0)x*
***************
*** 93,99 ****
  ...\glue 15.10124 plus -1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(12.38226+2.9229)x426.79135, glue set 181.27568fil
  ....\glue(\leftskip) 0.0 plus 1.0fil
  ....\TU/FandolHei(0)/m/n/16.06 附
--- 100,105 ----
***************
*** 115,121 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(14.05243+6.02255)x426.79135, glue set 120.64766fill
  ....\hbox(0.0+0.0)x0.0
  ....\rule(14.05243+6.02255)x0.0
--- 121,126 ----
***************
*** 196,202 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 84.51266fill
  ....\hbox(0.0+0.0)x0.0
--- 201,206 ----
***************
*** 284,290 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 107.06091fill
  ....\hbox(0.0+0.0)x0.0
--- 288,293 ----
***************
*** 370,376 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 108.60266fill
  ....\hbox(0.0+0.0)x0.0
--- 373,378 ----
***************
*** 454,460 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 200.3976fill
  ....\hbox(0.0+0.0)x0.0
--- 456,461 ----
***************
*** 520,526 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 204.96265fill
  ....\hbox(0.0+0.0)x0.0
--- 521,526 ----
***************
*** 588,594 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set - 0.04227
  ....\hbox(0.0+0.0)x0.0
--- 588,593 ----
***************
*** 701,707 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 50.38919fill
  ....\hbox(0.0+0.0)x0.0
--- 700,705 ----
***************
*** 803,809 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 168.82765fill
  ....\hbox(0.0+0.0)x0.0
--- 801,806 ----
***************
*** 877,883 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 156.78265fill
  ....\hbox(0.0+0.0)x0.0
--- 874,879 ----
***************
*** 953,959 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 132.69266fill
  ....\hbox(0.0+0.0)x0.0
--- 949,954 ----
***************
*** 1033,1039 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 217.00764fill
  ....\hbox(0.0+0.0)x0.0
--- 1028,1033 ----
***************
*** 1099,1105 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 72.46767fill
  ....\hbox(0.0+0.0)x0.0
--- 1093,1098 ----
***************
*** 1189,1195 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 120.64766fill
  ....\hbox(0.0+0.0)x0.0
--- 1182,1187 ----
***************
*** 1271,1277 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 156.78265fill
  ....\hbox(0.0+0.0)x0.0
--- 1263,1268 ----
***************
*** 1347,1353 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 126.00769fill
  ....\hbox(0.0+0.0)x0.0
--- 1338,1343 ----
***************
*** 1430,1436 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 114.39632fill
  ....\hbox(0.0+0.0)x0.0
--- 1420,1425 ----
***************
*** 1514,1520 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 139.57036fill
  ....\hbox(0.0+0.0)x0.0
--- 1503,1508 ----
***************
*** 1595,1601 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 229.05264fill
  ....\hbox(0.0+0.0)x0.0
--- 1583,1588 ----
***************
*** 1659,1665 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 107.26567fill
  ....\hbox(0.0+0.0)x0.0
--- 1646,1651 ----
***************
*** 1741,1747 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 124.17685fill
  ....\hbox(0.0+0.0)x0.0
--- 1727,1732 ----
***************
*** 1828,1834 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 229.05264fill
  ....\hbox(0.0+0.0)x0.0
--- 1813,1818 ----
***************
*** 1892,1898 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 103.43536fill
  ....\hbox(0.0+0.0)x0.0
--- 1876,1881 ----
***************
*** 1981,1987 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 228.46243fill
  ....\hbox(0.0+0.0)x0.0
--- 1964,1969 ----
***************
*** 2041,2047 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 186.89514fill
  ....\hbox(0.0+0.0)x0.0
--- 2023,2028 ----
***************
*** 2111,2117 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 175.52467fill
  ....\hbox(0.0+0.0)x0.0
--- 2092,2097 ----
***************
*** 2183,2189 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 186.89514fill
  ....\hbox(0.0+0.0)x0.0
--- 2163,2168 ----
***************
*** 2253,2259 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 125.33316fill
  ....\hbox(0.0+0.0)x0.0
--- 2232,2237 ----
***************
*** 2336,2342 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 126.67015fill
  ....\hbox(0.0+0.0)x0.0
--- 2314,2319 ----
***************
*** 2468,2473 ****
--- 2445,2451 ----
  ...\glue 0.0 plus 1.0fil minus 1.0fil
  .\kern 714.29128
  .\kern 0.0
+ .\kern 0.0
  )
  \tf@lot=\write...
  Completed box being shipped out [4]
***************
*** 2615,2621 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 210.98514fill
  ....\hbox(0.0+0.0)x0.0
--- 2593,2598 ----
***************
*** 2681,2687 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 174.85014fill
  ....\hbox(0.0+0.0)x0.0
--- 2658,2663 ----
***************
*** 2753,2759 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 210.98514fill
  ....\hbox(0.0+0.0)x0.0
--- 2729,2734 ----
***************
*** 2819,2825 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 36.33266fill
  ....\hbox(0.0+0.0)x0.0
--- 2794,2799 ----
***************
*** 2913,2919 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 84.51265fill
  ....\hbox(0.0+0.0)x0.0
--- 2887,2892 ----
***************
*** 2999,3005 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 132.69264fill
  ....\hbox(0.0+0.0)x0.0
--- 2972,2977 ----
***************
*** 3077,3083 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 84.51265fill
  ....\hbox(0.0+0.0)x0.0
--- 3049,3054 ----
***************
*** 3163,3169 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 45.69162fill
  ....\hbox(0.0+0.0)x0.0
--- 3134,3139 ----
***************
*** 3257,3263 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 45.69162fill
  ....\hbox(0.0+0.0)x0.0
--- 3227,3232 ----
***************
*** 3351,3357 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 229.05263fill
  ....\hbox(0.0+0.0)x0.0
--- 3320,3325 ----
***************
*** 3413,3419 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 122.98438fill
  ....\hbox(0.0+0.0)x0.0
--- 3381,3386 ----
***************
*** 3491,3497 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 14.6878fill
  ....\hbox(0.0+0.0)x0.0
--- 3458,3463 ----
***************
*** 3585,3591 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 0.19766fill
  ....\hbox(0.0+0.0)x0.0
--- 3551,3556 ----
***************
*** 3687,3693 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 192.91763fill
  ....\hbox(0.0+0.0)x0.0
--- 3652,3657 ----
***************
*** 3755,3761 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 8.89417fill
  ....\hbox(0.0+0.0)x0.0
--- 3719,3724 ----
***************
*** 3855,3861 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 71.62451fill
  ....\hbox(0.0+0.0)x0.0
--- 3818,3823 ----
***************
*** 3941,3947 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 42.35516fill
  ....\hbox(0.0+0.0)x0.0
--- 3903,3908 ----
***************
*** 4031,4040 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\penalty 10000
  ...\glue -6.0
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
--- 3992,4000 ----
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\penalty 10000
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -6.0
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
