*** ./build/test-testfiles/config-crossref/09-main-postdoc.tlg	2022-05-13 22:20:12.558133739 +0800
--- ./build/test-testfiles/config-crossref/09-main-postdoc.xetex.log	2022-05-13 22:20:17.138097555 +0800
***************
*** 16,21 ****
--- 16,23 ----
  Completed box being shipped out [1]
  \vbox(710.18088+4.1104)x439.87962
  .\hbox(0.0+0.0)x0.0
+ ..\special{pdfcolorstackinit 1 page direct (0 g 0 G)}
+ .\hbox(0.0+0.0)x0.0
  ..\kern -72.26999
  ..\vbox(0.0+0.0)x0.0, glue set 72.26999fil
  ...\kern -72.26999
***************
*** 79,86 ****
--- 81,93 ----
  ..\vbox(674.33032+0.0)x426.79135, glue set 214.01591fil
  ...\write-{}
  ...\special{color push gray 0}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnotehinttrue }{}}
+ ...\write1{\protect \providecommand {\protect \FN@pp@footnote@aux }[2]{}}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\glue(\topskip) 12.0
  ...\rule(0.0+0.0)x*
***************
*** 88,94 ****
  ...\glue 15.10124 plus -1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(12.3662+2.79442)x426.79135, glue set 189.30568fil
  ....\glue(\leftskip) 0.0 plus 1.0fil
  ....\TU/FandolHei(0)/m/n/16.06 摘
--- 95,100 ----
***************
*** 106,112 ****
  ...\mark{{摘\hskip 1em\relax 要}{摘\hskip 1em\relax 要}}
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(9.50351+2.32468)x426.79135, glue set - 0.37794
  ....\hbox(0.0+0.0)x24.09
  ....\TU/FandolSong(0)/m/n/12.045 本
--- 112,117 ----
***************
*** 353,359 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.58408
  ...\hbox(9.34692+2.10786)x426.79135, glue set 302.69174fil
  ....\hbox(0.0+0.0)x24.09
--- 358,363 ----
***************
*** 386,392 ****
  ...\glue 0.0 plus 1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.55998
  ...\hbox(9.40715+2.32468)x396.79135, glue set - 0.11517, shifted 30.0
  ....\hbox(8.14243+0.13248)x0.0
--- 390,395 ----
***************
*** 516,522 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.28296
  ...\hbox(9.50351+2.32468)x396.79135, glue set - 0.11517, shifted 30.0
  ....\hbox(8.14243+0.13248)x0.0
--- 519,524 ----
***************
*** 622,628 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.31909
  ...\hbox(9.636+2.6258)x396.79135, glue set - 0.13158, shifted 30.0
  ....\hbox(8.14243+0.16861)x0.0
--- 624,629 ----
***************
*** 723,729 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.295
  ...\hbox(9.50351+2.28853)x396.79135, glue set - 0.11517, shifted 30.0
  ....\hbox(8.14243+0.13248)x0.0
--- 724,729 ----
***************
*** 873,879 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.24681
  ...\hbox(9.50351+2.32468)x396.79135, glue set - 0.03839, shifted 30.0
  ....\hbox(8.28696+0.16861)x0.0
--- 873,878 ----
***************
*** 1007,1013 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.45158
  ...\hbox(9.50351+2.32468)x396.79135, glue set 91.94449fil, shifted 30.0
  ....\hbox(8.23878+0.16861)x0.0
--- 1006,1011 ----
***************
*** 1086,1092 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 8.24681
  ...\hbox(9.50351+2.28853)x396.79135, glue set - 0.01814, shifted 30.0
  ....\hbox(7.97379+0.13248)x0.0
--- 1084,1089 ----
***************
*** 1198,1204 ****
  ...\glue(\baselineskip) 17.9551
  ...\hbox(0.0+0.0)x0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 10.66785
  ...\hbox(9.40715+2.2283)x426.79135, glue set 134.70015fil
  ....\hbox(0.0+0.0)x24.09
--- 1195,1200 ----
***************
*** 1269,1278 ****
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue -2.2283
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
--- 1265,1273 ----
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -2.2283
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
***************
*** 1329,1334 ****
--- 1324,1330 ----
  ...\glue 0.0 plus 1.0fil minus 1.0fil
  .\kern 714.29128
  .\kern 0.0
+ .\kern 0.0
  Completed box being shipped out [2]
  \vbox(710.18088+4.1104)x439.87962
  .\hbox(0.0+0.0)x0.0
***************
*** 1390,1395 ****
--- 1386,1392 ----
  ..\glue(\lineskip) 0.0
  ..\vbox(674.33032+0.0)x426.79135, glue set 298.79561fil
  ...\write-{}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\glue(\topskip) 12.0
  ...\rule(0.0+0.0)x*
***************
*** 1397,1403 ****
  ...\glue 15.10124 plus -1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(11.70773+0.24089)x426.79135, glue set 183.9818fil
  ....\glue(\leftskip) 0.0 plus 1.0fil
  ....\TU/texgyreheros(0)/m/n/16.06 Abstract
--- 1394,1399 ----
***************
*** 1411,1417 ****
  ...\mark{{Abstract}{Abstract}}
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(8.22673+2.6258)x426.79135, glue set - 0.17136
  ....\hbox(0.0+0.0)x24.09
  ....\TU/texgyretermes(0)/m/n/12.045 Based
--- 1407,1412 ----
***************
*** 1703,1709 ****
  ...\glue 0.0 plus 1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.67966
  ...\hbox(8.22673+2.61375)x396.79135, glue set 0.2669, shifted 30.0
  ....\hbox(8.14243+0.13248)x0.0
--- 1698,1703 ----
***************
*** 1814,1820 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.22247
  ...\hbox(8.22673+2.6258)x396.79135, glue set 1.00089, shifted 30.0
  ....\hbox(8.14243+0.13248)x0.0
--- 1808,1813 ----
***************
*** 1913,1919 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.22247
  ...\hbox(8.22673+2.6258)x396.79135, glue set 157.69814fil, shifted 30.0
  ....\hbox(8.14243+0.16861)x0.0
--- 1906,1911 ----
***************
*** 1961,1967 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.22247
  ...\hbox(8.22673+2.6258)x396.79135, glue set 82.23625fil, shifted 30.0
  ....\hbox(8.14243+0.13248)x0.0
--- 1953,1958 ----
***************
*** 2017,2023 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.16225
  ...\hbox(8.28696+2.6258)x396.79135, glue set 182.0411fil, shifted 30.0
  ....\hbox(8.28696+0.16861)x0.0
--- 2008,2013 ----
***************
*** 2061,2067 ****
  ...\penalty -51
  ...\glue 0.0
  ...\glue(\parskip) 0.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 9.21042
  ...\hbox(8.23878+2.6258)x396.79135, glue set 105.12173fil, shifted 30.0
  ....\hbox(8.23878+0.16861)x0.0
--- 2051,2056 ----
***************
*** 2123,2129 ****
  ...\glue(\baselineskip) 17.4492
  ...\hbox(0.0+0.0)x0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 11.84827
  ...\hbox(8.22673+2.6258)x426.79135, glue set 66.96712fil
  ....\hbox(0.0+0.0)x24.09
--- 2112,2117 ----
***************
*** 2175,2184 ****
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\glue -2.6258
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
--- 2163,2171 ----
  ....\penalty 10000
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -2.6258
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
***************
*** 2301,2306 ****
--- 2288,2294 ----
  ..\glue(\lineskip) 0.0
  ..\vbox(674.33032+0.0)x426.79135, glue set >20000.0fil
  ...\write-{}
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\write-{}
  ...\glue(\topskip) 12.0
  ...\rule(0.0+0.0)x*
***************
*** 2308,2314 ****
  ...\glue 15.10124 plus -1.0
  ...\glue 0.0
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(12.43044+2.58565)x426.79135, glue set 181.27568fil
  ....\glue(\leftskip) 0.0 plus 1.0fil
  ....\TU/FandolHei(0)/m/n/16.06 目
--- 2296,2301 ----
***************
*** 2330,2336 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\hbox(14.05243+6.02255)x426.79135, glue set 251.79358fill
  ....\hbox(0.0+0.0)x0.0
  ....\rule(14.05243+6.02255)x0.0
--- 2317,2322 ----
***************
*** 2388,2394 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 358.53636fill
  ....\glue(\leftskip) 12.045
--- 2374,2379 ----
***************
*** 2428,2434 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 274.22137fill
  ....\glue(\leftskip) 12.045
--- 2413,2418 ----
***************
*** 2482,2488 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 289.27762fill
  ....\glue(\leftskip) 24.09
--- 2466,2471 ----
***************
*** 2530,2536 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 283.25513fill
  ....\glue(\leftskip) 24.09
--- 2513,2518 ----
***************
*** 2578,2584 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 283.25513fill
  ....\glue(\leftskip) 24.09
--- 2560,2565 ----
***************
*** 2626,2632 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 150.76015fill
  ....\glue(\leftskip) 24.09
--- 2607,2612 ----
***************
*** 2696,2702 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 304.33388fill
  ....\glue(\leftskip) 12.045
--- 2676,2681 ----
***************
*** 2744,2750 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 331.43512fill
  ....\glue(\leftskip) 24.09
--- 2723,2728 ----
***************
*** 2784,2790 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 259.16513fill
  ....\glue(\leftskip) 24.09
--- 2762,2767 ----
***************
*** 2836,2842 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 259.16513fill
  ....\glue(\leftskip) 24.09
--- 2813,2818 ----
***************
*** 2888,2894 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 247.12013fill
  ....\glue(\leftskip) 24.09
--- 2864,2869 ----
***************
*** 2942,2948 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 304.33388fill
  ....\glue(\leftskip) 12.045
--- 2917,2922 ----
***************
*** 2990,2996 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 331.43512fill
  ....\glue(\leftskip) 24.09
--- 2964,2969 ----
***************
*** 3030,3036 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 274.23343fill
  ....\glue(\leftskip) 24.09
--- 3003,3008 ----
***************
*** 3076,3082 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 229.70306fill
  ....\glue(\leftskip) 24.09
--- 3048,3053 ----
***************
*** 3128,3134 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 231.08824fill
  ....\glue(\leftskip) 24.09
--- 3099,3104 ----
***************
*** 3182,3188 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 223.03014fill
  ....\glue(\leftskip) 24.09
--- 3152,3157 ----
***************
*** 3240,3246 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 210.98514fill
  ....\glue(\leftskip) 24.09
--- 3209,3214 ----
***************
*** 3300,3306 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 307.34512fill
  ....\glue(\leftskip) 24.09
--- 3268,3273 ----
***************
*** 3344,3350 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 304.33388fill
  ....\glue(\leftskip) 12.045
--- 3311,3316 ----
***************
*** 3392,3398 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 162.80515fill
  ....\glue(\leftskip) 24.09
--- 3358,3363 ----
***************
*** 3460,3466 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 235.07513fill
  ....\glue(\leftskip) 24.09
--- 3425,3430 ----
***************
*** 3516,3522 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 259.16513fill
  ....\glue(\leftskip) 24.09
--- 3480,3485 ----
***************
*** 3568,3574 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 352.51387fill
  ....\glue(\leftskip) 12.045
--- 3531,3536 ----
***************
*** 3609,3615 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 221.68109fill
  ....\hbox(0.0+0.0)x0.0
--- 3571,3576 ----
***************
*** 3672,3678 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 352.51387fill
  ....\glue(\leftskip) 12.045
--- 3633,3638 ----
***************
*** 3712,3718 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 328.42387fill
  ....\glue(\leftskip) 12.045
--- 3672,3677 ----
***************
*** 3756,3762 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 256.15388fill
  ....\glue(\leftskip) 12.045
--- 3715,3720 ----
***************
*** 3984,3990 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 352.51387fill
  ....\glue(\leftskip) 12.045
--- 3942,3947 ----
***************
*** 4024,4030 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 316.37888fill
  ....\glue(\leftskip) 12.045
--- 3981,3986 ----
***************
*** 4070,4076 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 207.97389fill
  ....\glue(\leftskip) 12.045
--- 4026,4031 ----
***************
*** 4134,4140 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 223.03014fill
  ....\glue(\leftskip) 24.09
--- 4089,4094 ----
***************
*** 4192,4198 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 210.98514fill
  ....\glue(\leftskip) 24.09
--- 4146,4151 ----
***************
*** 4252,4258 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 223.03014fill
  ....\glue(\leftskip) 24.09
--- 4205,4210 ----
***************
*** 4310,4316 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 232.06389fill
  ....\glue(\leftskip) 12.045
--- 4262,4267 ----
***************
*** 4370,4376 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 207.97389fill
  ....\glue(\leftskip) 12.045
--- 4321,4326 ----
***************
*** 4434,4440 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 210.98514fill
  ....\glue(\leftskip) 24.09
--- 4384,4389 ----
***************
*** 4494,4500 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 265.18762fill
  ....\glue(\leftskip) 24.09
--- 4443,4448 ----
***************
*** 4544,4550 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 310.35637fill
  ....\glue(\leftskip) 12.045
--- 4492,4497 ----
***************
*** 4591,4597 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 263.83858fill
  ....\hbox(0.0+0.0)x0.0
--- 4538,4543 ----
***************
*** 4646,4652 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 346.49136fill
  ....\glue(\leftskip) 12.045
--- 4592,4597 ----
***************
*** 4686,4692 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 250.13138fill
  ....\glue(\leftskip) 12.045
--- 4631,4636 ----
***************
*** 4742,4748 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 250.13138fill
  ....\glue(\leftskip) 12.045
--- 4686,4691 ----
***************
*** 4798,4804 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 253.14262fill
  ....\glue(\leftskip) 24.09
--- 4741,4746 ----
***************
*** 4850,4856 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 217.00763fill
  ....\glue(\leftskip) 24.09
--- 4792,4797 ----
***************
*** 4908,4914 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 310.35637fill
  ....\glue(\leftskip) 12.045
--- 4849,4854 ----
***************
*** 4955,4961 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 191.56859fill
  ....\hbox(0.0+0.0)x0.0
--- 4895,4900 ----
***************
*** 5022,5028 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 346.49136fill
  ....\glue(\leftskip) 12.045
--- 4961,4966 ----
***************
*** 5062,5068 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 274.22137fill
  ....\glue(\leftskip) 12.045
--- 5000,5005 ----
***************
*** 5114,5120 ****
  ...\glue 0.0 plus 0.1
  ...\penalty 10000
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 274.22137fill
  ....\glue(\leftskip) 12.045
--- 5051,5056 ----
***************
*** 5167,5173 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 336.10857fill
  ....\hbox(0.0+0.0)x0.0
--- 5103,5108 ----
***************
*** 5211,5217 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 361.5476fill
  ....\hbox(0.0+0.0)x0.0
--- 5146,5151 ----
***************
*** 5251,5257 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 385.6376fill
  ....\hbox(0.0+0.0)x0.0
--- 5185,5190 ----
***************
*** 5287,5293 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 229.05263fill
  ....\hbox(0.0+0.0)x0.0
--- 5220,5225 ----
***************
*** 5351,5357 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 229.05263fill
  ....\hbox(0.0+0.0)x0.0
--- 5283,5288 ----
***************
*** 5415,5421 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 361.5476fill
  ....\hbox(0.0+0.0)x0.0
--- 5346,5351 ----
***************
*** 5455,5461 ****
  ...\penalty 10000
  ...\glue 6.02249
  ...\glue(\parskip) 0.0 plus 1.0
- ...\glue(\parskip) 0.0
  ...\glue(\baselineskip) 0.00002
  ...\hbox(14.05243+6.02255)x426.79135, glue set 337.45761fill
  ....\hbox(0.0+0.0)x0.0
--- 5385,5390 ----
***************
*** 5494,5503 ****
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\penalty 10000
  ...\glue -6.0
  ...\glue 0.0 plus 1.0fil
- ...\glue 0.0 plus -1.0fil
- ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
--- 5423,5431 ----
  ....\glue(\parfillskip) 0.0 plus 1.0fil
  ....\glue(\rightskip) 0.0
  ...\penalty 10000
+ ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue -6.0
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
  ..\glue(\baselineskip) 7.14894
