*** ./build/test/package-algorithms.tlg	2022-05-13 22:19:27.110491091 +0800
--- ./build/test/package-algorithms.xetex.log	2022-05-13 22:19:29.010476215 +0800
***************
*** 55,61 ****
  ....\kern 2.0
  ....\special{color push gray 0}
  ....\glue(\parskip) 0.0
- ....\glue(\parskip) 0.0
  ....\hbox(9.19034+2.20424)x426.79135
  .....\hbox(0.0+0.0)x0.0
  .....\glue 0.0
--- 55,60 ----
***************
*** 106,112 ****
  .....\penalty -51
  .....\glue 2.2082
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\hbox(7.58534+2.26344)x415.75012, glue set 318.3143fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
  ......\glue 0.0
--- 105,110 ----
***************
*** 148,154 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 4.2648
  .....\hbox(7.82538+2.02055)x415.75012, glue set 354.37411fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 146,151 ----
***************
*** 183,189 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 4.86919
  .....\hbox(7.46388+2.02055)x415.75012, glue set 388.3942fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 180,185 ----
***************
*** 212,218 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 4.70357
  .....\hbox(7.6295+0.26498)x415.75012, glue set 354.54172fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 208,213 ----
***************
*** 253,259 ****
  .....\glue 0.0
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.62476
  .....\hbox(7.46388+0.15457)x403.70512, glue set 363.29845fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 248,253 ----
***************
*** 285,291 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.98912
  .....\hbox(7.20993+0.08833)x403.70512, glue set 363.90572fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 279,284 ----
***************
*** 318,324 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.8014
  .....\hbox(7.46388+0.15456)x415.75012, glue set 395.82068fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 311,316 ----
***************
*** 345,351 ****
  .....\glue 0.0
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.98914
  .....\hbox(7.20993+0.09937)x403.70512, glue set 371.88855fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 337,342 ----
***************
*** 375,381 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 7.04433
  .....\hbox(7.20993+0.08833)x403.70512, glue set 371.46896fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 366,371 ----
***************
*** 407,413 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.63579
  .....\hbox(7.6295+0.15456)x415.75012, glue set 388.44511fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 397,402 ----
***************
*** 437,443 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.61372
  .....\hbox(7.58534+1.72243)x415.75012, glue set 341.04932fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 426,431 ----
***************
*** 479,485 ****
  .....\glue 0.0
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 5.0017
  .....\hbox(7.6295+0.15456)x403.70512, glue set 323.74438fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 467,472 ----
***************
*** 524,530 ****
  .....\glue 0.0
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.98914
  .....\hbox(7.20993+0.27603)x391.66013, glue set 335.17201fil, shifted 35.13123
  ......\hbox(0.0+0.0)x0.0
--- 511,516 ----
***************
*** 561,567 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.61371
  .....\hbox(7.46388+0.15457)x391.66013, glue set 345.93155fil, shifted 35.13123
  ......\hbox(0.0+0.0)x0.0
--- 547,552 ----
***************
*** 596,602 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.65788
  .....\hbox(7.54117+1.99844)x403.70512, glue set 330.57893fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 581,586 ----
***************
*** 637,643 ****
  .....\glue 0.0
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 5.14525
  .....\hbox(7.20993+2.02055)x391.66013, glue set 342.98921fil, shifted 35.13123
  ......\hbox(0.0+0.0)x0.0
--- 621,626 ----
***************
*** 672,678 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 4.86919
  .....\hbox(7.46388+0.0)x391.66013, glue set 336.53006fil, shifted 35.13123
  ......\hbox(0.0+0.0)x0.0
--- 655,660 ----
***************
*** 710,716 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.72412
  .....\hbox(7.6295+0.15456)x403.70512, glue set 376.40012fil, shifted 23.08623
  ......\hbox(0.0+0.0)x0.0
--- 692,697 ----
***************
*** 742,748 ****
  .....\penalty -51
  .....\glue 0.0
  .....\glue(\parskip) 0.0
- .....\glue(\parskip) 0.0
  .....\glue(\baselineskip) 6.61372
  .....\hbox(7.58534+0.15456)x415.75012, glue set 370.65767fil, shifted 11.04123
  ......\hbox(0.0+0.0)x0.0
--- 723,728 ----
***************
*** 779,786 ****
  ...\penalty 0
  ...\penalty 10000
  ...\glue 12.0 plus 2.0 minus 2.0
! ...\glue 0.0 plus 1.0fil
! ...\glue 0.0 plus -1.0fil
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
--- 759,765 ----
  ...\penalty 0
  ...\penalty 10000
  ...\glue 12.0 plus 2.0 minus 2.0
! ...\write1{\protect \FN@pp@footnotehinttrue }
  ...\glue 0.0 plus 1.0fil
  ...\glue 0.0
  ...\glue 0.0 plus 0.0001fil
